### ISSUE TITLE
Doesn't work on secure pages.

### ISSUE THREAD

**Issue by goodevilgenius** [allmarkedup/snoopy#14]

**Comment by michaelnordmeyer** [comment]
This fork does: https://github.com/michaelnordmeyer/snoopy

**Comment by goodevilgenius** [comment]
![image](https://cloud.githubusercontent.com/assets/254662/21352321/a4d675c0-c686-11e6-9040-8273581f764e.png)

Nope, that one doesn't work either.

Copied straight from `bookmarklet.js`, which has `http://...` in there a couple of times. Changing those to `https` fixed it.

**Comment by michaelnordmeyer** [comment]
You're right. I forgot to push the URL-encoded version which I did just now.

My fork is able to detect more services and uses upgraded libraries. Unfortunately my pull requests went unanswered so I keep it in a forked repo. ¯\(ツ)/¯
